docs/sources/alerting/unified-alerting/opt-in.md

```diff
@@ -19,38 +19,38 @@ To enable Grafana 8 alerts:
 
 Notification channels are migrated to an Alertmanager configuration with the appropriate routes and receivers. Default notification channels are added as contact points to the default route. Notification channels not associated with any Dashboard alert go to the `autogen-unlinked-channel-recv` route.
 
-Since `Hipchat` and `Sensu` notification channels are no longer supported, legacy alerts associated with these channels are not automatically migrated to Grafana 8 alerting. Assign the legacy alerts to a supported notification channel so that you continue to receive notifications for those alerts.
+Since `Hipchat` and `Sensu` notification channels are no longer supported, legacy alerts associated with these channels are not automatically migrated to Grafana alerting. Assign the legacy alerts to a supported notification channel so that you continue to receive notifications for those alerts.
 Silences (expiring after one year) are created for all paused dashboard alerts.
 
 ### Limitation
 
-Grafana 8 alerting system can retrieve rules from all available Prometheus, Loki, and Alertmanager data sources. It might not be able to fetch alerting rules from all other supported data sources at this time.
+Grafana alerting system can retrieve rules from all available Prometheus, Loki, and Alertmanager data sources. It might not be able to fetch alerting rules from all other supported data sources at this time.
 
-## Disable Grafana 8 alerts
+## Disable Grafana alerts
 
-To disable Grafana 8 alerts and enable legacy dashboard alerts:
+To disable Grafana alerts and enable legacy dashboard alerts:
 
-1. In your custom configuration file ($WORKING_DIR/conf/custom.ini), go to the [Grafana 8 alerting]({{< relref "../../administration/configuration.md#unified_alerting" >}}) section.
+1. In your custom configuration file ($WORKING_DIR/conf/custom.ini), go to the [Grafana alerting]({{< relref "../../administration/configuration.md#unified_alerting" >}}) section.
 1. Set the `enabled` property to `false`.
 1. For [legacy dashboard alerting]({{< relref "../../administration/configuration.md#alerting" >}}), set the `enabled` flag to `true`.
 1. Restart Grafana for the configuration changes to take effect.
 
-> **Note:** If you choose to migrate from Grafana 8 alerting to legacy dashboard alerting, you will lose any new alerts created in the Grafana 8 alerting system.
+> **Note:** If you choose to migrate from Grafana alerting to legacy dashboard alerting, you will lose any new alerts created in the Grafana alerting system.
```

Comment:
I think we need to highlight here the following situation: 
- user enables Grafana 8 alerting,
- the existing legacy alerts are migrated, 
- user adds\removes\edits alert rules
- user disables new alerting (or rolls back to legacy)
- user enables new alerting. In this case, the user will end up with either legacy alerts or an empty list of alerts (depending on whether the legacy alerts exist).

So, switching alerting back and forth can cause data loss. This is applicable to the fresh installation as well as upgraded ones.

Reply:
I have updated the note: Switching from one flavor of alerting to another can result in data loss. This is applicable to the fresh installation as well as upgraded setups.